### PR TITLE
Spring v3 , Converting to Spring Boot

### PR DESCRIPTION
**Pull request by Tolyaspit**
Please check changes and approve it)

### CODE REVIEW

File: src/main/java/com/andersen/course/TicketsApp.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import com.andersen.course.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import com.andersen.course.repository.TicketRepository;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+@SpringBootApplication
+@EntityScan(basePackages = {"com.andersen.course.model"})
```
Comment (Alexmansar) [inline src/main/java/com/andersen/course/TicketsApp.java:14]: could be in configuration class

File: src/main/java/com/andersen/course/config/Config.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.orm.hibernate5.HibernateTransactionManager;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@EnableTransactionManagement
+public class Config {
```
Comment (Alexmansar) [inline src/main/java/com/andersen/course/config/Config.java:17]: specify the name, like AppConfiguration

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Tolyaspit Hi, in general good job)) small comments))

File: src/main/java/com/andersen/course/config/Config.java (hunk 2 above)
Comment (Tolyaspit) [reply src/main/java/com/andersen/course/config/Config.java]: Like , leave the main method in the TicketApp and put the annotations to configuration class right?

File: src/main/java/com/andersen/course/TicketsApp.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import com.andersen.course.model.NewUser;
+import com.andersen.course.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import com.andersen.course.repository.TicketRepository;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+public class TicketsApp {
```
Comment (Alexmansar) [inline src/main/java/com/andersen/course/TicketsApp.java:14]: please, return SpringBootApplication

File: src/main/java/com/andersen/course/TicketsApp.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+    public static void main(String[] args) {
+        SpringApplication.run(AppConfiguration.class, args);
```
Comment (Alexmansar) [inline src/main/java/com/andersen/course/TicketsApp.java:17]: TicketsApp. class

File: src/main/java/com/andersen/course/config/AppConfiguration.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+@SpringBootApplication
```
Comment (Alexmansar) [inline src/main/java/com/andersen/course/config/AppConfiguration.java:14]: here remove))

**Review by Alexmansar — APPROVED** [review]
@Tolyaspit Hi)) olease, update code and merge it))

**Pull request merged**